extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +119,40 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
```

Comment:
Using window is not OK. `this.getWin()` is the right window. If `window` works, the player sends it to the wrong window.